doc/prompt_task_map.json

```diff
@@ -0,0 +1,746 @@
+{
+    "commonsense_qa": [
+        {
+            "task": "task073_commonsenseqa_answer_generation",
+            "prompt": [
+                "most_suitable_answer",
+				"question_answering",
+				"question_to_answer_index"
+            ]
+        }
+    ],
+    "dream": [
+        {
+            "task": "task247_dream_answer_generation",
+            "prompt": [
+                "baseline",
+				"read_the_following_conversation_and_answer_the_question"
+            ]
+        }
+    ],
+    "quail": [
+        {
+            "task": "task887_quail_answer_generation",
+            "prompt": [
+                "context_description_question_text",
+				"context_question_description_text",
+				"description_context_question_text"
+            ]
+        }
+    ],
+    "quartz": [
+    ],
+    "social_i_qa": [
+        {
+            "task": "task384_socialiqa_question_classification",
+            "prompt": [
+				"Check if a random answer is valid or not"
+            ]
+        },
+        {
+            "task": "task580_socialiqa_answer_generation",
+            "prompt": [
+				"Show choices and generate answer",
+				"Show choices and generate index"
+            ]
+        }
+    ],
+    "wiqa": [
+    ],
+    "cosmos_qa": [
+        {
+            "task": "task024_cosmosqa_answer_generation",
+            "prompt": [
+				"context_description_question_text",
+				"context_question_description_text",
+				"description_context_question_text"
+            ]
+        }
+    ],
+    "qasc": [
+    ],
+    "quarel": [
+        {
+            "task": "task1378_quarel_correct_answer_generation",
+            "prompt": [
+				"choose_between",
+				"do_not_use",
+				"heres_a_story",
+				"logic_test",
+				"testing_students"
+            ]
+        }
+    ],
+    "sciq": [
+        {
+            "task": "task591_sciq_answer_generation",
+            "prompt": [
+				"Direct Question (Closed Book)"
+            ]
+        }
+    ],
+    "wiki_hop/original": [
+    ],
+    "wiki_hop/masked": [
+    ],
+    "adversarial_qa/adversarialQA": [
```

Comment:
I think it's fine to keep all the four settings for `adversarial_qa` here. But when we add the missing tasks later, please just use this adversarialQA setting. Or, probably we can just drop the other three now to avoid confusion?

Reply:
I don't know the setting of our task and I'm not sure if it is `adversarial_qa` or not. 
If we don't need all the subsets, we can change the instances of this task and just keep `adversarial_qa`.